dristhi-app/src/main/java/org/ei/drishti/commonregistry/CommonObjectSort.java

```diff
@@ -0,0 +1,52 @@
+package org.ei.drishti.commonregistry;
+
+import org.ei.drishti.view.contract.SmartRegisterClient;
+import org.ei.drishti.view.contract.SmartRegisterClients;
+import org.ei.drishti.view.dialog.SortOption;
+
+import java.util.Collections;
+import java.util.Comparator;
+
+public class CommonObjectSort implements SortOption {
+    boolean byColumn;
+    boolean byDetails;
+    boolean isInteger;
+    String field;
+
+    public CommonObjectSort(boolean byColumn, boolean byDetails, boolean isInteger, String field) {
```

Comment:
I think it would be better if you using something like enum, so you can call this method like 
CommonObjectSort(CommonObjectSort.ByColumnAndByDetails, Field)